docs/topics/mta-rn-known-issues-7-0-1.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-1_{context}"]
+= Known issues
+
+{ProductFullName} version 7.0.1 has the following issues.
+
+.MTA CLI does not function on machines using ARM based architecture
```

Comment:
Shouldn't this also be `{ProductShortName}`?

Reply:
Changed